app/javascript/selectors/portfolioSelector.js

```diff
@@ -0,0 +1,28 @@
+export function getPortfolioName(portfolioId, state) {
+    let portfolios = state.portfolios; 
+    for (let key in portfolios) {
+      if (portfolios[key].id == portfolioId) {
+        return portfolios[key].name
```

Comment:
This function can just be written as
```
portfolios[portfolioId].name
```
Since portfolios are index by their id in the map, there's no need to loop through all of them to find a match, and this is faster